fineract-provider/src/main/java/org/apache/fineract/adhocquery/domain/AdHoc.java

```diff
@@ -35,7 +35,7 @@
 @Table(name = "m_adhoc")
 public class AdHoc extends AbstractAuditableCustom {
 
-      @Column(name = "name", length = 100)
+    @Column(name = "name", length = 100)
```

Comment:
avoid making formatting changes (this will likely be done as part of the ongoing Checkstyle work anyway)